src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -106,7 +112,25 @@
     @Argument(doc = "Stop after writing this many sequences.  For testing.")
     public int NUM_SEQUENCES = Integer.MAX_VALUE;
 
+    @Argument(shortName = "AN", doc = "Optional file containing the alternative names for the contigs. "
+            + "First column is the original name, the second column is an alternative name. "
+            + "One contig may have more than one alternative name. " ,
+            optional=true)
+    public File ALT_NAMES = null;
+
     private final MessageDigest md5;
+    
+    /** 
+     * Regular expression defined in the sam spec. Any alternative contig should match this regular expression 
+     * TODO: replace the pattern with a constant : see https://github.com/samtools/htsjdk/pull/956/files
+     */
+    private static final Pattern ALTERNATIVE_CONTIG_NAME_PATTERN = Pattern.compile("[0-9A-Za-z][0-9A-Za-z\\*\\+@\\|\\-]*");
```

Comment:
The `\\` characters in this string aren't necessary since the characters `*` `+` and `|` do not require quoting. `-` would require quoting but doesn't since it occurs last in the character class.